x-pack/legacy/plugins/reporting/server/routes/generate_from_savedobject_immediate.ts

```diff
@@ -51,15 +51,8 @@ export function registerGenerateCsvFromSavedObjectImmediate(
       const request = makeRequestFacade(legacyRequest);
       const logger = parentLogger.clone(['savedobject-csv']);
       const jobParams = getJobParamsFromRequest(request, { isImmediate: true });
-
-      /* TODO these functions should be made available in the export types registry:
```

Comment:
I think if we make the "execute job function factory" functions return immediately instead of returning a promise, I'd be satisfied this TODO is no longer needed.

Reply:
BTW a few more of these TODOs have been removed, above